src/styles/globals.css

```diff
@@ -26,6 +30,26 @@
   background: var(--overlay-1);
 }
 
+.icon-sol::before {
+  content: "◎";
+  @apply inline-block text-gray-500;
+}
+
+.prefix-input {
+  @apply relative;
+}
+.prefix-icon-sol:before {
+  content: "◎";
+  @apply inline-block absolute text-lg top-2 left-2 text-gray-500;
+}
+
+.wallet-adapter-button.wallet-adapter-button-trigger, .wallet-adapter-button:not([disabled]):hover {
+  @apply bg-white text-black hover:bg-gray-100 font-normal rounded-full h-14;
+
+}
```

Comment:
The modal buttons of picking a wallet provider get affected by this. Need to look at the user menu and picking provider according to the profile mocks.